syntax = "proto3";

message NoRequest {

}

message PersonMessage {
  string first_name = 1;
  string last_name = 2;
  string company_name = 3;
  int32 id = 4;
}

message PersonsMessage {
    repeated PersonMessage persons = 1;
}

service PersonService {
  rpc GetPersons(NoRequest) returns (PersonsMessage) {};
}